### ISSUE TITLE
Subtitle hack ->Soon new updated version of Jarvis? 

### ISSUE THREAD

**Issue by Marski69** [koying/SPMC#905]
http://blog.checkpoint.com/2017/05/23/hacked-in-translation/

**Comment by ghost** [comment]
https://github.com/xbmc/xbmc/pull/12024 hopefully we can see this fix also in our beloved Jarvis :)

**Comment by Skyrocket2Paradise** [comment]
I really hope so!

**Comment by TinderboxUK** [comment]
So this only affects those who streaming movies by the look of the video

**Comment by dkvdm** [comment]
Nope, you're exposed any time you're loading subtitles via a zip file, either by downloading them online via the built-in subtitle downloader or when you use the file browser to select your subtitle zip.
